README.md

```diff
@@ -24,3 +24,15 @@ Then, push each of the newly generated Docker images to the local registry with
     make push
 
 See https://github.com/docker/distribution/releases for the latest version of the registry server
+
+## Testing Images
+If you have created or modified an image, and have already created a local
+Docker repository as described above, perform the following steps to test your
+changes:
+
+1. `make <imagename>`
+1. `make push <imagename>`
```

Comment:
Is this step necessary for testing an image @bchavet ?

Reply:
Not the `make push`, no. after `make <imagename>` it already exists in your local docker images as `localhost:5000/imagename`

Reply:
Just for my own knowledge, when _is_ `make push` useful/needed?

Reply:
I use it to deploy to our production registry.